android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletConstants.java

```diff
@@ -120,11 +123,16 @@ public final class WalletConstants {
     public static final String SOL_INS_SIG_VERIFY = "Ed25519SigVerify111111111111111111111111111";
     public static final String SOL_INS_SECP = "KeccakSecp256k11111111111111111111111111111";
 
-    public static final List<String> KNOWN_TEST_CHAIN_IDS = Arrays.asList(
-            BraveWalletConstants.GOERLI_CHAIN_ID, BraveWalletConstants.SEPOLIA_CHAIN_ID,
-            BraveWalletConstants.LOCALHOST_CHAIN_ID, BraveWalletConstants.SOLANA_TESTNET,
-            BraveWalletConstants.SOLANA_DEVNET, BraveWalletConstants.FILECOIN_TESTNET,
-            BraveWalletConstants.FILECOIN_ETHEREUM_TESTNET_CHAIN_ID);
+    public static final Map<String, Integer> KNOWN_TEST_CHAINS_MAP = Map.of(
```

Comment:
Map.of is for API 30+ so it will crash on below 30 APIs. I once created `NATIVE_TOKENS_PER_CHAIN` map for aurora mapping, can use the same.

Reply:
Nice catch!
Thank you, I'll change it!